connector_jira_servicedesk/models/jira_backend/common.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2016-2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class JiraBackend(models.Model):
+    _inherit = "jira.backend"
+
+    organization_ids = fields.One2many(
+        comodel_name="jira.organization",
+        inverse_name="backend_id",
+        string="Organizations",
+        readonly=True,
+    )
+
+    organization_field_name = fields.Char(
+        string="Organization Field",
+        help="The 'Organization' field on JIRA is a custom field. "
+        "The name of the field is something like 'customfield_10002'. ",
+    )
+
+    @api.model
+    def _selection_project_template(self):
+        selection = super()._selection_project_template()
+        selection += [
+            ("Basic", "Basic (Service Desk)"),
+            ("IT Service Desk", "IT Service Desk (Service Desk)"),
+            ("Customer service", "Customer Service (Service Desk)"),
+        ]
+        return selection
+
+    def import_organization(self):
+        self.env["jira.organization"].import_batch(self)
```

Comment:
@sonhd91 it should be `self.env["jira.organization"].with_delay(channel="root.connector_jira.import").import_batch(self)`, because we've already removed the decorator of the given job